### ISSUE TITLE
Feature: add ZeroMQ support forwarding incoming transactions like original bitcoind?

### ISSUE THREAD

**Issue by typoworx-de** [spesmilo/electrum#6235]
Is there any plan to support ZeroMQ forwarding incoming transactions like original bitcoind?

**Comment by ecdsa** [comment]
not really. we have a 'notify' command on the CLI that you can use, but I agree that it would be better to have a proper notification interface
